### PR TITLE
Lambda function storage check

### PR DESCRIPTION
**Pull request by mgreschl**
addresses #19

### CODE REVIEW

File: aws_quota/check/__init__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from .elasticbeanstalk import *
 from .elb import *
 from .iam import *
+from .lambdas import *
```
Comment (mgreschl) [inline aws_quota/check/__init__.py:13]: I originally named this `lambda` but that's a reserved word and it didn't work. I'm open to suggestions for the name of this file.

Comment (brennerm) [reply aws_quota/check/__init__.py:13]: Fine for me :+1:

**Comment by brennerm** [comment]
Looking good :+1: 
Do you think there will be other checks that you may add in the near future or should I create another release today? :)

**Comment by mgreschl** [comment]
> Looking good 👍
> Do you think there will be other checks that you may add in the near future or should I create another release today? :)

Nothing in the near future. Thanks!

**Pull request merged**